app/forms/arp_payments_form.rb

```diff
@@ -0,0 +1,47 @@
+class ArpPaymentsForm < QuestionsForm
+  set_attributes_for :intake,
+                     :received_stimulus_payment,
+                     :eip1_amount_received,
+                     :eip2_amount_received,
+                     :eip3_amount_received,
+                     :advance_ctc_amount_received,
+                     :received_advance_ctc_payment
+  validate :eip_amounts_present_or_unsure
+  validate :advance_ctc_amount_present_or_unsure
+
+  def save
+    intake.assign_attributes(attributes_for(:intake))
+
+    unless received_stimulus_payment == "unsure"
```

Comment:
just noting that as written this will still accept/persist any numerical values the client enters even when they also check `unsure`

we could make it a truly exclusive thing (provide numbers *or* leave them blank and click 'unsure') but I guess that's up to whatever design wants

Reply:
its fine that they add numbers and check unsure -- they provide a value, but they're not quite sure. This indicates to the tax preparer that they may need to reconfirm those numbers with the client on the intake call.